docs/website/contents/for-developers/ChainSync.md

```diff
@@ -1,10 +1,12 @@
 # Chain sync client specification
 
+The aim of the chain sync client is to replicate the chain of an upstream node. For every upstream node that the local node is connected to there is a separate instance of the client. In the following, we consider a single instance.
```

Comment:
Line wrapping at ~80 chars?

Reply:
Not sure if we strictly follow that though

Reply:
@nfrisby made a point for not wrapping at all when editing markdown files. I think he prefers one line per sentence.

Reply:
Let's resolve then